src/Sentry/SentryTransaction.cs

```diff
@@ -339,6 +339,7 @@ public void AddBreadcrumb(Breadcrumb breadcrumb) =>
         _breadcrumbs.Add(breadcrumb);
 
     /// <inheritdoc />
+    [Obsolete("Add metadata to Contexts.Trace.SetData")]
```

Comment:
Should we be doing this? I thought Extra was supposed to stay the way it is... or is this something you've seen it the docs or other SDKs?

Reply:
Not sure what the answer is here?   @bruno-garcia

Reply:
Deprecate extra and add `SetData` to `Span` and `Transaction` please.